includes/class-sensei-course-progress-widget.php

```diff
@@ -195,7 +195,8 @@ function widget( $args, $instance ) {
 			}
 		}
 
-		echo $before_widget; ?>
+		echo $before_widget; // phpcs:ignore WordPress.XSS.EscapeOutput.OutputNotEscaped
```

Comment:
To be extra safe we usually run anything that's supposed to be unescaped HTML through `wp_kses_post`.